### PR TITLE
D8CORE-1852 Setup node revision delete on stanford_page

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Setup node revision delete module and configuration

# Need Review By (Date)
- 4/29

# Urgency
- medium

# Steps to Test
1. `composer require su-sws/stanford_profile:dev-D8CORE-1852`
1. `drush cim -y`
1. create a node and edit it at least 5 times to generate the different revisions
1. `drush cron`
1. verify you are left with 5 revisions on the node

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -112,6 +112,7 @@
         "drupal/menu_link_weight": "^1.0-beta3",
         "drupal/menu_position": "~1.0@alpha",
         "drupal/metatag": "~1.3",
+        "drupal/node_revision_delete": "^1.0@RC",
```
Comment (sherakama) [inline composer.json:115]: caps needed?

File: config/sync/core.entity_view_display.node.stanford_page.default.yml [hunk 2]
```diff
@@ -27,6 +27,7 @@ third_party_settings:
         layout_settings:
           extra_classes: ''
           centered: null
+          columns: default
```
Comment (sherakama) [inline config/sync/core.entity_view_display.node.stanford_page.default.yml:30]: This isn't really part of this update but good to have. I know what this is about.

File: tests/codeception/acceptance/Contrib/NodeRevisionDeleteCest.php [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<?php
+
+class NodeRevisionDeleteCest {
+
+  public function testNodeRevisionDelete(AcceptanceTester $I) {
```
Comment (sherakama) [inline tests/codeception/acceptance/Contrib/NodeRevisionDeleteCest.php:5]: Perhaps a docblock comment on what we are testing for?

File: composer.json (hunk 1 above)
Comment (pookmish) [reply composer.json:115]: :shrug: its what composer gave me when i did `composer require drupal/node_revision_delete --no-update`

**Pull request merged**